docs/reference/workflows/monobehaviour/creating-workers.md

```diff
@@ -84,69 +87,14 @@ When inheriting from the [`WorkerConnector`]({{urlRoot}}/api/core/worker-connect
 `GetReceptionistConfig` to modify the connection configuration a worker type uses to connect to the
 Locator or Receptionist. See [Connecting to SpatialOS]({{urlRoot}}/reference/concepts/connection-flows) to find out more about the connection flows for client-workers and server-workers.
 
-**Example** </br>
-Overriding the Receptionist connection flow configuration.
-
-```csharp
-protected override ReceptionistConfig GetReceptionistConfig(string workerType)
-{
-    return new ReceptionistConfig
-    {
-        WorkerType = workerType,
-        WorkerId = CreateNewWorkerId(workerType),
-        UseExternalIp = UseExternalIp
-    };
-}
-```
-
 ## How to decide which connect service to use
 
 First, see [Connecting to SpatialOS]({{urlRoot}}/reference/concepts/connection-flows) to find out more about the connection flows for client-workers and server-workers).
 
-The [`WorkerConnector`]({{urlRoot}}/api/core/worker-connector) provides a default implementation which automatically decides which connection flow to use based on whether the application is running in your Unity Editor and whether it can find a login token. You can change the behavior by overriding the `GetConnectionService` method.
-
-**Example** </br>
-Overriding which connection service to choose.
-
-```csharp
-protected override ConnectionService GetConnectionService()
-{
-    // Your worker will always connect via the Receptionist.
-    // This is the expected behavior for any server-worker.
-    return ConnectionService.Receptionist;
-}
-```
+The [`DefaultWorkerConnector`]({{urlRoot}}/api/core/default-worker-connector) provides a default implementation on how to decide which connection flow to use based on the received command line arguments. You can change the behavior by overriding the `GetConnectionService` method.
```

Comment:
where/when are the command line arguments received? (I know, but would the average user know how to find/change/tweak these if they want) (Do they need to?) (just thinking aloud)

Reply:
good point. will add some explanation that those are defined in the worker config and are used by the launcher